doc-content/jbpm-docs/src/main/asciidoc/BPMN2/dmn-execution-business-process.adoc

```diff
@@ -57,6 +57,8 @@ image::dmn/dmn-execution-business-process.png[]
 * *Namespace*: Enter the unique namespace from the DMN model file. Example: `\https://www.drools.org/kie-dmn`
 * *Decision Name*: Enter the name of the DMN decision node that you want to invoke in the selected process node. Example: `Rail`
 * *DMN Model Name*: Enter the DMN model name. Example: `Compute Rail`
++
+IMPORTANT: When you explore the root node, ensure that the *Namespace* and *DMN Model Name* fields consists of the same value. 
```

Comment:
If they have to be same, then we should also provide same examples, right? As right now it's `https://www.drools.org/kie-dmn` and `Compute Rail`.

Reply:
@MarianMacik, thanks for raising this. The idea is that the value of Namespace and DMN Model Name should be same in BPMN as DMN diagram. To make it clearer, how about changing this to:

```suggestion
IMPORTANT: When you explore the root node, ensure that the *Namespace* and *DMN Model Name* fields consist of the same value in BPMN as DMN diagram. 
```

Reply:
I see now, thanks for clarification. Yes, you can go ahead with that note.

Reply:
@MarianMacik, Done. Can you please take a look and approve the changes?